From 35ec818fb2eb1e029368102c9f0a6e2c6244141e Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Sat, 9 May 2015 17:59:26 +0100
Subject: [PATCH] p2v: Update documentation for Configure network button.

This updates commit e464774a79ff9eeedcdb3fd53efd5a7898ca036e.

(cherry picked from commit 61a021713e8a93faeadc093e5c2d5064ba86059c)
---
 p2v/virt-p2v.pod | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/p2v/virt-p2v.pod b/p2v/virt-p2v.pod
index 744e8d4..15e68bd 100644
--- a/p2v/virt-p2v.pod
+++ b/p2v/virt-p2v.pod
@@ -114,7 +114,7 @@ At the bottom of the dialog are these buttons:
  │                                                        │
  │                [ Test connection ]                     │
  │                                                        │
- │                [ About           ]  [ Next           ] │
+ │ [ Configure network ] [ About virt-p2v  ] [ Next     ] │
  │                                                        │
  └────────────────────────────────────────────────────────┘
 
@@ -124,6 +124,10 @@ have supplied the correct server name, user name, password, etc., and
 a suitable version of virt-v2v is available remotely) then press the
 C<Next> button to move to the next dialog.
 
+You can use the C<Configure network> button if you need to assign a
+static IP address to the physical machine, or use Wifi, bonding or
+other network features.
+
 =head2 DISK AND NETWORK CONFIGURATION DIALOG
 
 The second configuration dialog lets you configure the details of
-- 
1.8.3.1

